packages/TermsAndConditions/StyledClickable.jsx

```diff
@@ -1,9 +1,9 @@
 import styled from 'styled-components'
 
```

Comment:
Just wondering if this abstraction is still needed if it's only used in one place: https://github.com/telus/tds-core/blob/e39b3c0dbf2ce622f5ef038aec4d6aa791d8f9e0/packages/TermsAndConditions/svg/Chevron.jsx#L14

Reply:
Fixed.